package.json

```diff
@@ -24,10 +24,10 @@
     "lib"
   ],
   "engines": {
-    "node": ">= 0.6"
+    "node": ">= 0.12"
   },
   "dependencies": {
-    "css-select": "^3.1.2",
+    "cheerio-select": "github:cheeriojs/cheerio-select",
```

Comment:
We have a Github reference here, until https://github.com/cheeriojs/cheerio-select/issues/2 is resolved.